packages/api/lambdas/index-from-database.js

```diff
@@ -71,7 +71,7 @@ async function indexModel({
 
 async function indexFromDatabase(event) {
   const {
-    index: esIndex,
+    indexName: esIndex,
```

Comment:
Great catch. Epic fail on my part

Reply:
Also something that TS would catch if we had a description for the typing of `event`, but I digress